Cargo.lock

```diff
@@ -1637,13 +1825,13 @@ dependencies = [
 
 [[package]]
 name = "quinn-proto"
-version = "0.10.6"
+version = "0.11.1"
```

Comment:
A `cargo update -p quinn-proto` will bump this to 0.11.2, which will bring in https://github.com/quinn-rs/quinn/pull/1876 and fix the build err in `crates/proto/src/error.rs`